### PR TITLE
Implemented database and reports 

### PR DESCRIPTION
**Pull request by Glipsy**
I did the work please review and let me know if anything needs to be changed

### CODE REVIEW

File: 1 tables.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+create database TrackGermDB
```
Comment (jacobglanz) [inline 1 tables.sql:1]: @Glipsy I was unable to run the file more then once, i think you may be missing a `drop database statement`.

File: 1 tables.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+go
+use TrackGermDB
+go
+create table dbo.Germ(
```
Comment (jacobglanz) [inline 1 tables.sql:5]: missing a `primary key`

File: 1 tables.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+	GermName varchar(200) not null,
+	Host varchar(20)
```
Comment (jacobglanz) [inline 1 tables.sql:7]: Should not allow `null`, also we need to check constraint as is in the spec

**Review by jacobglanz — CHANGES_REQUESTED** [review]
I reviewed the work it was pretty good, I noticed a few thing missing, please see my inline comments

File: 2 sample data.sql [hunk 4]
```diff
@@ -0,0 +1,5 @@
+select 'Norovirus', 'human'
```
[the remark this answers is not in the document]
Comment (Glipsy) [reply 2 sample data.sql:4]: added the insert statement

**Review by Glipsy — COMMENTED** [review]
Thanks for the feedback i addressed all the issues you raised, see my latest comments

File: 1 tables.sql (hunk 1 above)
Comment (jacobglanz) [reply 1 tables.sql:5]: Its not working i think you misspelled something

File: 1 tables.sql (hunk 2 above)
Comment (jacobglanz) [reply 1 tables.sql:9]: good

Comment (jacobglanz) [reply 1 tables.sql:9]: All good

File: 2 sample data.sql (hunk 4 above)
[the remark this answers is not in the document]
Comment (jacobglanz) [reply 2 sample data.sql:4]: missing `use `statement

[the remark this answers is not in the document]
Comment (jacobglanz) [reply 2 sample data.sql:4]: this insert only inserted one row... is something missing?

**Review by jacobglanz — CHANGES_REQUESTED** [review]

**Review by jacobglanz — APPROVED** [review]

**Pull request closed without merging**